Sources/Sentry/SentryBreadcrumbTracker.m

```diff
@@ -141,13 +141,32 @@ - (void)addEnabledCrumb
     [SentrySDK addBreadcrumb:crumb];
 }
 
-- (void)swizzleSendAction
++ (BOOL)avoidSender:(id)sender forTarget:(id)target action:(NSString *)action
```

Comment:
`m`: If you put `#if SENTRY_HAS_UIKIT` around the whole function you don't need to add the return false below.